src/ng/compile.js

```diff
@@ -1386,6 +1386,65 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var commentDirectivesEnabled = true;
+  /**
+   * @ngdoc method
+   * @name $compileProvider#commentDirectivesEnabled
+   * @description
+   *
+   * It indicates to the compiler
+   * whether or not directives on comments should be compiled.
+   * Defaults to `true`.
+   *
+   * Calling this function with false disables the compilation of directives
+   * on comments for the whole application.
+   * This results in a compilation performance gain,
+   * as the compiler doesn't have to check comments when looking for directives.
+   * This should however only be used if you are sure that no comment directives are used in the application
+   * (including any 3rd party directives).
+   *
+   * Example:
+   *
+   * ```
+   * $compileProvider.commentDirectivesEnabled(false);
+   * ```
+   *
+   * @param {boolean} false if the compiler may ignore directives on comments
+   */
+  this.commentDirectivesEnabled = function(value) {
```

Comment:
We usually make these configuration function getter/setter (and the getters return `self` for chaining).
It makes sense to keep consistent with that.